app/services/authorizer.rb

```diff
@@ -1,19 +1,23 @@
 class Authorizer
+  include AuthorizerCache
```

Comment:
Why is this extra module? I don't see it being included anywhere else. If you want to extract it from `Authorizer` to keep the object responsibility simple, I'd recommend extracting it to a separate class under Authorizer name scope.